### PR TITLE
fix: Switch merchant enabled for Live Env

### PR DESCRIPTION
**Pull request by PritishBudhiraja**
## Type of Change

- [x] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Checklist

- [x] I ran `npm run re:build`
- [x] I reviewed submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

File: src/screens/HyperSwitch/SelfServe/HSwitchProdOnboarding/HSwitchAgreementScreen.res [hunk 1]
```diff
@@ -101,8 +117,11 @@ let make = () => {
               buttonType={Primary}
               buttonSize={Small}
               customButtonStyle="!px-2 rounded-lg"
-              onClick={_ => downloadPDF()}
-              buttonState={isSelected && isAgreeEnable ? Normal : Disabled}
+              onClick={_ =>
+                GlobalVars.isLocalhost
+                  ? setDashboardPageState(_ => #PROD_ONBOARDING)
```
Comment (JeevaRamu0104) [inline src/screens/HyperSwitch/SelfServe/HSwitchProdOnboarding/HSwitchAgreementScreen.res:122]: remove this condition

File: src/screens/HyperSwitch/SwitchMerchant/SwitchMerchant.res [hunk 2]
```diff
@@ -170,24 +170,26 @@ module ExternalUser = {
...
-                            `${activeClasses} text-blue-900 flex gap-2 font-medium w-56`
-                          }>
-                          <Icon name="plus-circle" size=15 />
-                          {"Add a new merchant"->React.string}
-                        </button>}
-                    </Menu.Item>
-                  </div>
+                  <UIUtils.RenderIf condition={isAddMerchantEnabled}>
+                    <div className="px-1 py-1 ">
+                      <Menu.Item>
```
Comment (JeevaRamu0104) [inline src/screens/HyperSwitch/SwitchMerchant/SwitchMerchant.res:175]: make this has a separate module

File: src/screens/HyperSwitch/SelfServe/HSwitchProdOnboarding/HSwitchAgreementScreen.res (hunk 1 above)
Comment (PritishBudhiraja) [reply src/screens/HyperSwitch/SelfServe/HSwitchProdOnboarding/HSwitchAgreementScreen.res]: addressed

File: src/screens/HyperSwitch/SwitchMerchant/SwitchMerchant.res (hunk 2 above)
Comment (PritishBudhiraja) [reply src/screens/HyperSwitch/SwitchMerchant/SwitchMerchant.res]: Addressed

**Review by JeevaRamu0104 — APPROVED** [review]

**Pull request merged**